www/artworks/new.php

```diff
@@ -0,0 +1,144 @@
+<?php /** @noinspection PhpUndefinedMethodInspection,PhpIncludeInspection */
+require_once('Core.php');
+
+session_start();
+
+$exception = $_SESSION['exception'] ?? null;
+
+if ($exception) {
+	http_response_code(422);
+	session_unset();
+}
+
+$success = $_SESSION['successfully-submitted-artwork'] ?? null;
+
+if ($success) {
+	http_response_code(201);
+	session_unset();
+}
+
+?>
+<?= Template::Header(
+	[
+		'title' => 'Submit Artwork',
+		'highlight' => '',
+		'description' => 'Submit public domain artwork to the database for use as cover art.'
+	]
+) ?>
+<main>
+	<section class="narrow">
+		<hgroup>
+			<h1>Submit Artwork</h1>
+			<h2></h2>
+		</hgroup>
+
+		<?= Template::Error(['exception' => $exception]) ?>
+
+		<? if ($success) { ?>
+			<p class="message success">
+				Thank you for submitting artwork. It will be reviewed and added to the database if it is approved.
+			</p>
+		<? } ?>
+
+		<form method="post" action="/artworks" enctype="multipart/form-data">
+			<fieldset>
+				<legend>Artist Details</legend>
+				<div>
+					<label>
+						Artist Name
+						<datalist id="artist-names">
```

Comment:
I agree that it needs refinement - what if we eventually have works by 10,000 artists, are we going to include them all in the form html? Hopefully not.

I suppose the most straightforward way to achieve autofill for the death year field would be an accompanying JS script that retrieves that data from a `data-death-year` attribute on the artist in the datalist. We would want to get Alex's OK on that first though as there is no JS elsewhere in the site. In this case it might be alright as it wouldn't be providing critical functionality -- users with scripts disabled would just be missing autofill for one field, which isn't the end of the world.

Reply:
It's not an issue of scale, just convenience and correctness. 

Let me think about this some more and maybe prototype something. I was imagining a form where the user would either pick an existing artist (both name and death year), or enter both manually. What we have now is a mix where the user picks the name but must always enter the death year manually.

No Javascript is a [core principle](https://alexcabal.com/posts/standard-ebooks-and-classic-web-tech), so I would prefer not to add it here.

Reply:
Cool - let's make a note on the main issue to handle this better at some point down the road.